river/command.zig

```diff
@@ -69,6 +69,9 @@ const command_impls = std.ComptimeStringMap(
         .{ "map",                       @import("command/map.zig").map },
         .{ "map-pointer",               @import("command/map.zig").mapPointer },
         .{ "map-switch",                @import("command/map.zig").mapSwitch },
+        // NOTE: `set-main-count` should only be called by the layout manager and not directly.
+        // Advertises the number of views in the main area to river.
```

Comment:
```suggestion
```

There is absolutely no reason users should not use this command directly. In fact I suspect layout generators calling it will be the minority.